src/registrar/health-id-otp-generation/health-id-otp-generation.component.ts

```diff
@@ -456,11 +527,21 @@ export class HealthIdOtpSuccessComponent implements OnInit, DoCheck {
   ) {
     console.log('popupdata');
   }
-  succdata: any = this.data.data;
+  succdata: any = this.data;
+  txnId: any = this.data.txnId
   ngOnInit() {
     this.assignSelectedLanguage();
     this.fetchHealthIdsValue();
     console.log('popupdata', this.succdata);
+    if(this.succdata.newAbhaResponse != undefined && this.succdata.newAbhaResponse != null){
+      this.abhaProfileData = this.succdata.newAbhaResponse;
+      console.log("abha mobile number -", this.abhaProfileData?.ABHAProfile?.mobile)
+      if(this.abhaProfileData?.ABHAProfile?.mobile != null && this.abhaProfileData?.ABHAProfile?.mobile != undefined){
```

Comment:
Pls use "!==" instead of "!="

Reply:
This file will be deleted in next PR as i have refactored all the abdm components